spec/models/question_advice_spec.rb

```
@@ -0,0 +1,118 @@
+    describe '#advice' do
+        it 'returns the score of QuestionAdvice' do
+            expect(question_advice1.advice).to eq('default advice')
+            expect(question_advice2.advice).to eq('advice for question 2')
+            expect(question_advice3.advice).to eq('advice for question 3')
+        end
+        
+    end
+
+    describe '#test export_fields' do
+        it 'make sure that the columns of QuestionAdvice are properly being mapped' do
+            output = QuestionAdvice.export_fields(options = {})
```

RuboCop (1.86.1)

[convention] Use 2 (not 4) spaces for indentation.

(Layout/IndentationWidth)